core/cog-webkit-utils.c

```diff
@@ -19,11 +19,13 @@ static const char error_message_template[] =
     "        margin-left: 0.75em; margin-top: 0 }\n"
     ".try-again { text-align: center; font-size: 1em; \n"
     "             height: 100%; margin: 1em; }\n"
-    "</style></head><body>\n"
+    "</style>\n"
+    "<script>\nfunction retry() { window.location.href = '%s' }\n"
+    "setTimeout(retry, 5000);\n</script></head><body>\n"
```

Comment:
This make the “retry every five seconds” behaviour unconditional, and I can imagine there may be cases in which this is not desired. It would be better if we could have an option to configure (or disable) the retry interval.

Reply:
You're right, and option would be best. Should it be disabled by default?

Reply:
Yes, disabled by default would keep the current behaviour by default, to avoid breaking people's setups. Thanks for considering reworking the patch 😸